pandas/tests/indexing/test_loc.py

```diff
@@ -119,15 +119,15 @@ def test_loc_getitem_label_out_of_range(self):
                           typs=['ints', 'uints', 'labels', 'mixed', 'ts'],
                           fails=KeyError)
         self.check_result('label range', 'loc', 'f', 'ix', 'f',
-                          typs=['floats'], fails=TypeError)
+                          typs=['floats'], fails=KeyError)
```

Comment:
is this reflected in the whatsnew?

Reply:
This is not a change in the API: I just fixed the test, which was checking nothing at all (because test objects for floats were missing).